meson.build

```diff
@@ -0,0 +1,56 @@
+project(
+    'python-rtmidi',
+    'cpp',
+    version: '1.5.0rc1',
+    license: 'MIT',
+    default_options: [
+        'warning_level=2'
+    ],
+    meson_version: '>=0.53.0'
+)
+
+# Dependencies
+cpp = meson.get_compiler('cpp')
+jack2_dep = dependency('jack', version:'>=1.9.11', required: false)
+jack1_dep = dependency('jack', version:'>=0.125.0, <1.0', required: false)
+alsa_dep = dependency('alsa', required: false)
+thread_dep = dependency('threads')
+coremidi_dep = dependency('threads', required: false)
+winmm_dep = dependency('threads', required: false)
+
+if not jack2_dep.found() and jack1_dep.found()
+    jack_dep = jack1_dep
+elif jack2_dep.found()
+    jack_dep = jack2_dep
+else
+    warning('No version of JACK found, which is recent enough (jack2>=1.9.11 or jack1>=0.125.0)')
+endif
+
+pymod = import('python')
+python = pymod.find_installation(
+    get_option('python'),
+    required: true
+)
+
+# Installation directories
+prefix = get_option('prefix')
+bindir = join_paths(prefix, get_option('bindir'))
+
+# Build jack_mix_box and generate _jack_mixer extension source
+subdir('src')
+
+# Build & install C extension module and Python package
+subdir('rtmidi')
+
+if not get_option('wheel')
+    meson.add_install_script('meson_postinstall.py')
+endif
+
+summary({
+    'Debug messages (verbose)': get_option('verbose'),
+    'Build for wheel': get_option('wheel'),
+    'ALSA support': target_machine.system() == 'linux' and alsa_dep.found() and get_option('alsa'),
```

Comment:
Note target_machine is intended when doing a https://en.wikipedia.org/wiki/Cross_compiler#Canadian_Cross build of a compiler, and you want to specify the machine code format the cross-compiled binaries will do codegen for.

For most cases you want host_machine, which is the system that you're compiling for (either the current machine, or when cross compiling, the version you are cross compiling for).

Reply:
Fixed in aa28305.